### DISCUSSION TITLE
SQLAlchemy 2 RC is on PyPI. Please help by testing on your project!

### DISCUSSION THREAD

**Discussion by CaselIT** [sqlalchemy/sqlalchemy#8632] [Ideas]
Version 2 brings a lot of changes, and it would be very helpful if as many people as possible test the beta to help in fixing issues before the official release.

You can install with `pip install SQLAlchemy==2.0.0rc2`

What's new: https://docs.sqlalchemy.org/en/20/changelog/whatsnew_20.html
Migration from 1.4: https://docs.sqlalchemy.org/en/20/changelog/migration_20.html
Changelog: https://docs.sqlalchemy.org/en/20/changelog/changelog_20.html

**Comment by mcclurem** [comment]
I'm extremely excited about 2.0 and starting a new project currently debating between 1.4 and 2.0.0beta. Is there an official target release date for non-beta?

**Reply by zzzeek** [reply]
it would be nice if it was January, sometime.   beta4 is very overdue and there's 50/50 we will do a beta5, then again, this really has to go out.

that's as "official" as I can get! ;)

**Comment by CaselIT** [comment]
v2 is out, unpinning this

thanks to anyone who tried the beta and rc versions!
